### PR TITLE
Basic Authentication

### LINKED ISSUE #46: Research and Develop User Authentication Feature
Using Auth0.

Page to modify: /login.js

### PR DESCRIPTION
**Pull request by xzz-wang**
#PR: 
Related Issues: Tag your issue on the right panel of github page after you submit your PR
Author @xzz-wang 

Description: You can successfully log into the system. The api works with backend.

### CODE REVIEW

**Comment by xzz-wang** [comment]
I am hoping to merge this after the styles are merged into the main branch.

**Comment by xzz-wang** [comment]
That was intense

**Review by lmy0125 — APPROVED** [review]
everything is good at this point

**Pull request merged**
